scripts/copy-bundle.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+
+set -e
+
+usage() {
+  echo "Usage: $0
+  [ -i BUNDLE_IMAGE a pull spec to a bundle image to copy ]
+  [ -n PKG_NAME the new operator package name ]
+  [ -v BUNDLE_VERSION the new operator version ]
+  [ -r OCP_RANGE the new supported OpenShift version range ]
```

Comment:
A bit more info on what format the range is expected?

Reply:
Added a link to the OCP docs.